client/src/components/outfitGeneratorList.jsx

```diff
@@ -1,9 +1,19 @@
 import React from 'react';
-import outfitGeneratorListEntry from './outfitGeneratorListEntry.jsx';
-
-function OutfitGeneratorList(props) {
-  
-}
+import OutfitGeneratorListEntry from './outfitGeneratorListEntry.jsx';
 
+const OutfitGeneratorList = (props) => (
+  <div id="list">
+    <h3>{props.style}</h3>
+    {
+      props.outfits.map((outfit) => {
+        return (
```

Comment:
I think you can use () instead of {} since it's one return statement. 
props.outfits.map(outfit => (<div>.....</div>));